src/browser/stacktrace/utils.ts

```
@@ -181,7 +181,7 @@ export const filterExtraStackFrames = (error: Error): Error => {
         };
 
         const isWdioUtilsFrame = (frame: StackFrame): boolean => {
-            return Boolean(frame.fileName && frame.fileName.includes("/node_modules/@wdio/utils/"));
+            return Boolean(frame.fileName && frame.fileName.includes("/node_modules/@testplane/wdio-utils/"));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)